tests/behaviour/expectations/behaviour.ts

```diff
@@ -2737,6 +2737,10 @@ export const expectations = flatten(
             Expect.Simple('bitwiseOr', [], ['22']),
             Expect.Simple('bitwiseXor', [], ['20']),
           ]),
+          File.Simple('booleanOperation', [
+            Expect.Simple('false_and', [], ['1']),
```

Comment:
I don't understand why the expected value here is `1`. If `callNull()` returns `false`, then `callNull() && true` should return `false` as well. If this test is passing there is probably something wrong with the code.

Reply:
The bug was because the boolean expression was being replaced by the incorrect member. Once that was fixed another errors arose due to ast inconsistencies.